scripts/lib/mlkem_test.py

```diff
@@ -272,23 +291,16 @@ def __init__(self, opts: Options):
         copts = CompileOptions(
             opts.cross_prefix, opts.cflags, opts.arch_flags, opts.auto, opts.verbose
         )
+
+        self._f = lambda t: Test_Implementations(t, copts)
```

Comment:
Is this change merely meant as a unification of L278-285? The behaviour is different because `_f` creates an object upon each invocation, while L278-L285 create objects only once.

Reply:
your right, this is not ideal, and not necessary